src/link/C/zig.h

```diff
@@ -26,6 +26,16 @@
 #define ZIG_RESTRICT
 #endif
 
+#if __STDC_VERSION__ >= 201112L
+#include <stdalign.h>
+#define ZIG_ALIGN(alignment) alignas(alignment)
+#elif defined(__GNUC__)
+#define ZIG_ALIGN(alignment) __attribute__((aligned(alignment)))
+#else
+/* TODO: Can we somehow align stack variables without help from the compiler? */
+#define ZIG_ALIGN(alignment)
```

Comment:
Here's an idea:

```suggestion
#define ZIG_ALIGN(alignment) zig_compile_error("the C compiler being used does not support aligning variables")
```

And then don't define `zig_compile_error` anywhere (that's how it will be a compile error).

What do you think?

Reply:
Sounds good to me.